### DISCUSSION TITLE
开启openclash后xbox主机的nat总是不通。

### DISCUSSION THREAD

**Discussion by daowaiyaoyao** [vernesong/OpenClash#3743] [Q&A]
只要开着openclash，我xbox游戏主机nat就是无法使用状态。 然后关闭openclash，nat就变成开放了。  每次打游戏之前都得手动关掉openclash，好麻烦。  有解决办法吗？
![Screenshot_20240129_132439_Xbox](https://github.com/vernesong/OpenClash/assets/58784225/a35f8c04-46a0-4e19-bfeb-0d6b63c4f782)

**Comment by ghost** [comment]
先确定节点udp是通的
```url
https://github.com/vernesong/OpenClash/discussions/3684
https://github.com/vernesong/OpenClash/issues/1997
https://github.com/vernesong/OpenClash/issues/2288
```
虽然好像也没完全解决，先跟着试试吧
